torchani/data/new.py

```diff
@@ -273,14 +313,27 @@ def ShuffledDataset(file_path,
     def my_collate_fn(data, chunk_threshold=chunk_threshold):
         return collate_fn(data, chunk_threshold)
 
-    data_loader = torch.utils.data.DataLoader(dataset=dataset,
-                                              batch_size=batch_size,
-                                              shuffle=shuffle,
-                                              num_workers=num_workers,
-                                              pin_memory=False,
-                                              collate_fn=my_collate_fn)
-
-    return data_loader
+    val_size = int(validation_split * len(dataset))
+    train_size = len(dataset) - val_size
+    train_dataset, val_dataset = torch.utils.data.random_split(dataset, [train_size, val_size])
+
+    train_data_loader = torch.utils.data.DataLoader(dataset=train_dataset,
+                                                    batch_size=batch_size,
+                                                    shuffle=shuffle,
+                                                    num_workers=num_workers,
+                                                    pin_memory=False,
+                                                    collate_fn=my_collate_fn)
+    if val_size == 0:
+        return train_data_loader
+
+    val_data_loader = torch.utils.data.DataLoader(dataset=val_dataset,
+                                                  batch_size=batch_size,
+                                                  shuffle=shuffle,
```

Comment:
Do we need to shuffle the validation set?

Reply:
Theoretically, there should be no difference. But no shuffle might take less computation, I will change it to `False`